v2/devices/axolotl.yml

```diff
@@ -4,38 +4,29 @@ formfactor: "phone"
 aliases: []
 doppelgangers: []
 user_actions:
-  confirm_model:
-    title: "Confirm your model"
-    description: "Please check that your device is a SHIFT6mq (axolotl)."
-  confirm_os:
-    title: "Confirm OS version"
-    description: "Please double-check that your device is running the latest available stock Android 10 firmware (type: release)."
-    link: "https://downloads.shiftphones.com/axolotl"
-  unlock_bootloader:
-    title: "Unlock your phone"
-    description: 'Your device must be "unlocked" in order to be able to flash it. Unlocking the bootloader will delete your personal data!<br><br>If your bootloader is already unlocked, you are good to go. Otherwise please check the link below (More...) and follow the instructions.'
-    link: "https://wiki.lineageos.org/devices/axolotl/install#unlocking-the-bootloader"
   bootloader:
     title: "Reboot to Bootloader"
     description: "With the device powered off, hold Volume Up + Power."
     image: "phone_power_up"
     button: true
+  fastbootd:
+    title: "Reboot to Fastbootd"
+    description: "With the device powered off, hold Volume Down + Power to boot into recovery.\n\nInside the recovery, press on 'Advanced' and select 'Enter fastboot'."
```

Comment:
This step is generic for Halium 10 devices.
Should we implement that common somewhere?

I think we should try to reduce user visible text in device configs as much as possible to make it easier for translation.